.github/workflow_scripts/run_benchmark.sh

```diff
@@ -20,3 +20,10 @@ python CI/bench/evaluate.py --config_path ./ag_bench_runs/tabular/ --time_limit
 aws s3 cp --recursive ./results s3://autogluon-ci-benchmark/cleaned/$BRANCH_OR_PR_NUMBER/$SHA/
 aws s3 rm --recursive s3://autogluon-ci-benchmark/cleaned/$BRANCH_OR_PR_NUMBER/latest/
 aws s3 cp --recursive ./results s3://autogluon-ci-benchmark/cleaned/$BRANCH_OR_PR_NUMBER/latest/
+
+cwd=`pwd`
+ls data/results/output/openml/ag_eval/pairwise/* | grep .csv > $cwd/agg_csv.txt
+filename=`head -1 $cwd/agg_csv.txt`
+prefix=$BRANCH_OR_PR_NUMBER/$SHA
+agdash --per_dataset_csv  'data/results/output/openml/ag_eval/results_ranked_by_dataset_all.csv' --agg_dataset_csv $filename --s3_prefix $prefix --s3_bucket ag-dashboard-test --s3_region us-east-2 > $cwd/out.txt
```

Comment:
We need to update the bucket and the region before merging. We can sync offline on this